testing/btest/Baseline/broker.store.brokerstore-backend-sqlite-corrupt-delete/out

```diff
@@ -0,0 +1,5 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+store is open
+populated 100 rows
+store is open
+populated 100 rows
```

Comment:
Do you think it's worth verifying in the test that these post-delete writes actually made it into the database?

Reply:
Sure.